css/index.css

```diff
@@ -1,3 +1,416 @@
+  margin-top: 35px;
+  align-items: center;
+  
+}
+
+header nav a {
+  
+  margin-right: 65px;
+  font-size: 14px;
+  color: gray;
+  text-decoration: none;
+  height: 40px;
+  
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 25px;
+  align-items: center;
+}
+
+/* cta */
+
+.cta{
+	display:flex;
+	
+}
+
+.cta .cta-text {
+  vertical-align: top;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  
+  margin-top: 69px;
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  padding: 0 10px;
+  display: flex;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+ display: flex;
+flex-direction: column;
+  margin-right: 34px;
+ 
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  margin-top: 37px;
+  padding: 0 10px;
+display: flex;
+
+}
+
+.main-content .bottom-content .text-content {
+  width: 275px;
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+
+.main-content .top-row{
+	display: flex;
+	width: 875px;
+	align-content: space-between;
+}
+
+.main-content .second-row{
+	display: flex;
+	width: 875px;
+	align-content: space-between;
+	
+}
+
+.main-content .third-row{
+	display: flex;
+	width: 875px;
+	align-content: space-between;
+	
+}
+
+.box{
+	border: 1px solid black;
+	background: lightgray;
+	padding: 20px;
+	margin: 10px;
+}
+.box button{
+	padding:5px;
+	height: 25px;
+	width: 100px;
+	margin:10px;
+}
+.bottom-info {
+	display: flex;
+	border-top: 1px solid black;
+	padding: 30px 0;
+}
+ .main-content{
+	 padding: 20px 0;
+ }
+.services-header-img{
+	padding: 15px 0;
+}
+.top-content{
+	padding: 15px 0 0 0;
+}
+.top-content h1{
+	font-size: 25px;
+}
+
+.bottom-info ul{
+	padding:0 0 0 20px;
+}
+
+/* Tablet View */
+@media (max-width:800px) {
+
+.middle-img{
+  width: 100%;
+  height: auto;
+}
+
+.cta img{
+  display: none;
+}
+
+.cta{
+  display: flex;
+  justify-content: center;
+}
+
+header{
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: center;
+} 
+}
+
+
+/* Mobile View */
+@media (max-width:500px) {
+
+  .cta img{
```

Comment:
No need to repeat that rule for all the breakpoints. If it was already declared for "max width: 800px" it will behave like this for everything below that. CSS is also cascading so whatever was already declared at the top will take effect, so it is wise to add only rules that You wish to CHANGE in certain breakpoints as there is no reason to repeat them.

That way we also adhere to DRY principles.